.github/action_helper.py

```diff
@@ -65,9 +66,10 @@ def setup_action(input_: str) -> None:
     if len(versions) > 20:
         raise ValueError(f"too many interpreters to install: {len(versions)} > 20")
 
-    print(f"::set-output name=interpreter_count::{len(versions)}")
-    for i, version in enumerate(versions):
-        print(f"::set-output name=interpreter_{i}::{version}")
+    with open(os.environ["GITHUB_OUTPUT"], "a") as fh:
```

Comment:
I was thinking it could just print the output, then this could be run as `python action_helper.py > $GITHUB_OUTPUT`. Might be a bit easier to test, too?

Reply:
Good point, that way the test doesn't need the fixture that monkeypatches the `GITHUB_OUTPUT` variable, and the risk of the test accidentally setting an output if there's a bug is gone. :smiley_cat:

Reply:
And updated.